test_sets/policies_all_flows.txt

```diff
@@ -402,7 +402,6 @@
 ./test_scripts/Policies/Validation_of_PolicyTables/296_ATF_Validate_preDataConsent_and_stealFocus_true.lua
 ./test_scripts/Policies/Validation_of_PolicyTables/297_ATF_Validate_preDataConsent_and_keepContext_false.lua
 ./test_scripts/Policies/Validation_of_PolicyTables/298_ATF_Validate_preDataConsent_and_keepContext_true.lua
-./test_scripts/Policies/Validation_of_PolicyTables/299_ATF_Memory_Kb_Constraints_Ignoring.lua
```

Comment:
It appears there is a similar test name in `test_sets/Deleted_from_CI.txt`, perhaps that should be removed as well?

Reply:
yeah, it's possible we could remove the whole set at some point, but I don't think it's a high priority in any case